### PR TITLE
clarify that the `chunk_size` is optional when streaming to a file

### PR DESCRIPTION
**Pull request by afeld**
Tripped me up briefly...figured others might run into the same thing. Thanks!

### CODE REVIEW

File: docs/user/quickstart.rst [hunk 1]
```diff
@@ -184,7 +184,7 @@ streamed to a file::
 Using ``Response.iter_content`` will handle a lot of what you would otherwise
 have to handle when using ``Response.raw`` directly. When streaming a
 download, the above is the preferred and recommended way to retrieve the
-content.
+content. Note that `chunk_size` is optional.
```
Comment (Lukasa) [inline docs/user/quickstart.rst]: In restructuredtext you need to use two backticks, as elsewhere in the paragraph. Can you amend to do that please?

**Comment by nateprewitt** [comment]
It's probably important to note that you _should_ provide a chunk_size here despite it being optional. The default of 1 is likely not what the user wants for any non-trivial file.

**Comment by afeld** [comment]
@nateprewitt Yeah, thought about that...might make sense to give a value of `128` or something reasonable in the example, and just say that it can be changed. I'm no expert on this, so whatever y'all think!

**Comment by nateprewitt** [comment]
Great! How does changing the example to:

```python
with open(filename, 'wb') as fd:
    for chunk in r.iter_content(chunk_size=128):
        fd.write(chunk)
```
and then that line saying you can adjust the chunk_size to a number that fits your requirements sound? This will probably better serve the user than pointing them towards the default behaviour of iter_content. That's just one guys thoughts though :)

**Comment by Lukasa** [comment]
I'd be happy to merge that docs change.

**Comment by nateprewitt** [comment]
@afeld if you think this change is agreeable and have the inclination, feel free to throw the PR together. Otherwise, I'll try to open something next week.

**Pull request merged**